lib/loader/index.js

```diff
@@ -135,6 +186,16 @@ module.exports = {
       var firstScript = getFirstScript();
 
       firstScript.parentNode.insertBefore(script, firstScript);
+
+      _loadedUrls[url] = true;
+
+      if (options.namespaceName) {
+        namespace.loadedUrls = _loadedUrls;
```

Comment:
As I understand it, by calling `registerProperty`, you are establishing ownership over that property. Multiple apps can read it, but only one should call `registerProperty` and that is the only one that should be allowed to modify it. Correct?